docs/user/document/aggregate_severity/text-usage.en.md

```diff
@@ -1 +1,8 @@
 # Text - Usage
+
+Represents overall assessment and should be a manufacturer-specific enum.
+
+*Examples:*
+
+* Assessment according to the German Advisory Format
+* Rating between 1 and 5
```

Comment:
```suggestion
* Assessment according to the German Advisory Format (DAF)
* Rating between 1 and 5
* Red Hat security ratings
* SSVC decision or vector
```
Some links here - I'm unsure yet how to incorporate them (maybe in the examples of the namespace?):
1. [www.cert-verbund.de/daf/](http://www.cert-verbund.de/daf/) => Careful, currently dead link => https://web.archive.org/web/20100921073502/http://www.cert-verbund.de/daf/daf_description.html
2. ?
3. https://access.redhat.com/security/updates/classification/
4. https://github.com/CERTCC/SSVC

Reply:
I opted for making the list items link to the resources you mentioned, except the first one that I skipped. Should we really use a link from archive.org? Maybe we can open a ticket to remember to add the link to cert-verbund.de once it's there (I understand the _currently_ as that it will become available in the future)

Also, I changed the 3rd bullet point to _"Red Hat Severity Ratings"_ as that's the title there.

Please let me know, if this works for you.

Reply:
Works for me. Please do so.

Reply:
As discussed, I changed the ordering to:

1. SSVC
2. Red Hat Severity Ratings
3. DAF
4. Rating 1 - 5